pegasus/sites.v3/code.org/layouts/default.haml

```diff
@@ -66,8 +66,13 @@
 
 -if header['theme'] == 'responsive'
   .desktop-feature
+    -if header['footer'] == 'suppl_footer'
```

Comment:
I'm going to change it to donor_footer instead, and it will only add the donor_slider if specified. Petition is going to be part of the regular page. It needs to be hidden for users < 13 anyway.